src/RemoteFramesProvider.js

```diff
@@ -73,6 +73,7 @@ class RemoteFramesProvider extends Component {
       <GlobalTarget
         onAddStackElement={this.props.onFrameAdded}
         onEmptyStack={this.props.onNoFrames}
+        onRemoveStackElement={this.props.onFrameRemoved}
```

Comment:
why don't we keep same name :D onRemoveStackElement to onFrameRemoved :/

Reply:
I keep the wording and naming identical to the existing handlers.